examples/snippets/interleaved-tables/README.md

```diff
@@ -32,62 +31,55 @@ CREATE TABLE singers (
 ) PRIMARY KEY (singerid);
 
 CREATE TABLE albums (
-    albumid INT64 NOT NULL,
     singerid INT64 NOT NULL,
+    albumid INT64 NOT NULL,
     title STRING(MAX)
 ) PRIMARY KEY (singerid, albumid), INTERLEAVE IN PARENT singers;
 
 CREATE TABLE tracks (
-    trackid INT64 NOT NULL,
     singerid INT64 NOT NULL,
     albumid INT64 NOT NULL,
+    trackid INT64 NOT NULL,
```

Comment:
same here. Why do we move `trackid`?

Reply:
Same as with `albumid`, Ruby on Rails 7.1 generates the columns in a different order than the third-party `composite-pk` gem.